requirements.txt

```
@@ -1,3 +1,4 @@
 pyTooling >= 5.0.0
 pyVHDLModel >= 0.27.1
 pySVModel>=0.3.5
+pySystemRDLModel>=0.0.1
```

Rerun pipeline by accepting this change :).
```suggestion
pySystemRDLModel >= 0.1.0
```